ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -58,6 +58,7 @@
 workloads = pytest.mark.workloads
 flowtests = pytest.mark.flowtests
 performance = pytest.mark.performance
+performance_extended = pytest.mark.performance_extended
```

Comment:
This new marker should also be defined here:
https://github.com/red-hat-storage/ocs-ci/blob/master/pytest.ini#L2

Reply:
@petr-balogh   
Added ( line 24) 

23    performance: performance related tests
24    performance_extended: non regression performance tests
25   scale: scale related tests